modules/network-observability-quickfilter.adoc

```diff
@@ -13,6 +13,12 @@ You can use *Query Options* to optimize the search results, as listed below:
 ** *Log Type*: The available options *Conversation* and *Flows* provide the ability to query flows by log type, such as flow log, new conversation, completed conversation, and a heartbeat, which is a periodic record with updates for long conversations. A conversation is an aggregation of flows between the same peers.  
 ** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for the *Overview* and *Topology* view. The default selected value is *Destination*.
 ** *Match filters*: You can determine the relation between different filter parameters selected in the advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
+** *Packet loss*: You can view different levels of dropped packets with the following query options:
+*** *Dropped* shows flow records with fully dropped packets.
```

Comment:
the whole line is bold I think which should not be the case

Reply:
<img width="398" alt="image" src="https://github.com/openshift/openshift-docs/assets/77501157/1e948596-93f2-488c-aff9-8feb16fe9483">

I think its just bolded because it renders as a different bullet format..not sure but the preview is OK and as expected.